front/scripts/main/components/DiscussionEditorComponent.ts

```
@@ -27,26 +29,36 @@ App.DiscussionEditorComponent = Em.Component.extend(App.ViewportMixin, {
 	style: Em.computed('isSticky', function (): string {
 		return this.get('isSticky') === true
 			? `height: ${this.$('.editor-container').outerHeight(true)}px`
-			: null;
+		: null;
 	}),
 
+	getBreakpointHeight(): number {
+		return this.offsetTop - (this.get('siteHeadPinned') ? this.siteHeadHeight : 0);
+	},
+
 	/**
-	 * Initialize onScroll binding for sticky logic
+	 * Method should be overwritten in the child classes
 	 * @returns {void}
 	 */
-	initializeOnScroll: Em.on('didInsertElement', function (): void {
-		this.offsetTop = this.$().offset().top;
-		this.siteHeadHeight = Em.$('.site-head').outerHeight(true);
+	initializeStickyState(): void {},
```

if it should be overwritten, it should throw an exception